charts/teuto-portal-k8s-worker/templates/_helpers.tpl

```diff
@@ -0,0 +1,12 @@
+{{- define "portalworker.image" -}}
+{{ include "common.images.image" (dict "imageRoot" .Values.worker.image "global" .Values.global) }}
+{{- end -}}
+
+{{- define "portalworker.jdbc" -}}
+{{- $port := .Values.worker.database.port | int -}}
+{{ printf "jdbc:postgresql://%s:%d/teuto_domain?currentSchema=app_public" .Values.worker.database.host $port }}
+{{- end -}}
+
+{{- define "portalworker.dbcredentials" -}}
```

Comment:
I would rename this to something like `portalworker.dbCredentials.secretName` to be more explicit

Reply:
renamed